cluster-autoscaler/processors/provreq/provisioning_request_processor.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // ProvisioningRequestProcessor process ProvisioningRequests in the cluster.
 type ProvisioningRequestProcessor interface {
-	Process([]*provreqwrapper.ProvisioningRequest)
+	Process(*provreqclient.ProvisioningRequestClient, []*provreqwrapper.ProvisioningRequest)
```

Comment:
Not clearly related to this PR, but I think all processors are generally expected to take AutoscalingContext as first parameter.